### PR TITLE
Arabic Updates

### PR DESCRIPTION
**Pull request by ghost**
Thanks @LaneSh4d0w @Nintendocustom for the repl link

### CODE REVIEW

File: channels/ae.m3u [hunk 1]
```diff
@@ -31,6 +31,8 @@ https://admdn4ta.cdn.mgmlcdn.com/adsports4/smil:adsports4.stream.smil/playlist.m
 https://admdn2.cdn.mangomolo.com/adtv/smil:adtv.stream.smil/chunklist.m3u8
 #EXTINF:-1 tvg-id="AEHawasTV.ae" tvg-name="AE | Hawas TV" tvg-country="AE" tvg-language="" tvg-logo="" group-title="",AE | Hawas TV
 https://jmc-live.ercdn.net/hawastvhd/hawastvhd_720p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://www.alaan.tv/wp-content/themes/alaan-child-tv/assets/img/logo.png" group-title="",Al Aan (720p)
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/video/x74wje5
```
Comment (Dum4G) [inline channels/ae.m3u]: I can't open any of @LaneSh4d0w links in VLC, does it work?

Comment (LaneSh4d0w) [reply channels/ae.m3u]: Hmm... Let me see if it works.

Comment (LaneSh4d0w) [reply channels/ae.m3u]: It was down. Try again.

**Comment by tilfizyawn** [comment]
I switched to https://iptv--iptv.repl.co/streamlink?url=https://www.dailymotion.com/video/x74wje5 because https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/video/x74wje5 stopped working but it's back now

**Comment by LaneSh4d0w** [comment]
> 
> 
> I switched to https://iptv--iptv.repl.co/streamlink?url=https://www.dailymotion.com/video/x74wje5 because https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/video/x74wje5 stopped working but it's back now

Yeah my server went down for no reason. It should be back up.

**Comment by tilfizyawn** [comment]
Add both mirrors

**Review by heroku-miraheze — APPROVED** [review]
Done.

**Review by freearhey — APPROVED** [review]

**Pull request merged**